full_utility_rates/California/Huntington Beach City of - 270/hbc-2017-03-04.owrs

```diff
@@ -0,0 +1,85 @@
+---
```

Comment:
Also the Utility ID is 277

the directory should be named "Huntington Beach City of - 277"